applications/AdjointFluidApplication/custom_schemes/adjoint_bossak_scheme.h

```diff
@@ -19,15 +19,18 @@
 #include "includes/define.h"
 #include "includes/element.h"
 #include "includes/condition.h"
+#include "includes/communicator.h"
 #include "includes/model_part.h"
 #include "includes/process_info.h"
 #include "includes/kratos_parameters.h"
 #include "includes/ublas_interface.h"
 #include "utilities/openmp_utils.h"
 #include "solving_strategies/schemes/scheme.h"
+#include "containers/variable.h"
+#include "includes/kratos_components.h"
 
 // Application includes
-#include "custom_utilities/objective_function.h"
+#include "../../AdjointFluidApplication/custom_utilities/objective_function.h"
 
```

Comment:
This relative import should not be necessary... do you need it because you are also using the file from trilinos? (otherwise I would try to avoid it)